src/main/webapp/WEB-INF/tags/instructor/results/byParticipantQuestionParticipant.tag

```diff
@@ -0,0 +1,19 @@
+<%@ tag description="instructorFeedbackResults - by Giver > Question > Recipient or Recipient > Question > Giver" %>
+<%@ tag import="teammates.common.util.Const" %>
+
+<%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c" %>
+<%@ taglib prefix="fn" uri="http://java.sun.com/jsp/jstl/functions" %>
+<%@ taglib tagdir="/WEB-INF/tags/instructor/results" prefix="results" %>
+
+<%@ attribute name="showAll" type="java.lang.Boolean" required="true" %>
+<%@ attribute name="shouldCollapsed" type="java.lang.Boolean" required="true" %>
+
+<br>
+<c:set var="teamIndex" value="${0}"/>
```

Comment:
`0` should be recognised just as well as `${0}`.
I believe it would be converted into an int naturally if you use it in a mathematical operation.